kitchen.dokken.yml

```diff
@@ -16,11 +16,6 @@ provisioner:
   multiple_converge: 2
 
 platforms:
-  - name: amazonlinux-2
-    driver:
-      image: dokken/amazonlinux-2
-      pid_one_command: /usr/lib/systemd/systemd
-
```

Comment:
Why is this being removed?

Reply:
It isnt listed as supported in the metadata so I removed it